### PR TITLE
TE: New UI/email template

### PR DESCRIPTION
**Pull request by puneetjaiswal**

### CODE REVIEW

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/bao/MergedAnomalyResultManager.java [hunk 1]
```diff
@@ -42,5 +42,7 @@ MergedAnomalyResultDTO findLatestConflictByFunctionIdDimensions(Long functionId,
 
   List<MergedAnomalyResultDTO> findByTime(long startTime, long endTime);
 
+  List<MergedAnomalyResultDTO> findUnNotifiedByFunctionIdAndIdLesserThanAndEndTimeGreaterThanLastOneDay(long functionId, long anomalyId);
```
Comment (apucher) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/bao/MergedAnomalyResultManager.java]: It's hard to tell what this function is for. I assume it's related to sending out email summaries. Is there a better way of naming it?

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/v2/AlertTaskRunnerV2.java [hunk 2]
```diff
@@ -105,10 +105,15 @@ private void sendAnomalyReport() throws Exception {
...
         }
-        // TODO: @pjaiswal / @ychang
         // fetch anomalies having id lesser than the watermark for the same function with notified = false & endTime > last one day
         // these anomalies are the ones that did not qualify filtration rule and got modified.
         // We should add these again so that these could be included in email if qualified through filtration rule
+        List<MergedAnomalyResultDTO> filteredAnomalies = anomalyMergedResultDAO
+            .findUnNotifiedByFunctionIdAndIdLesserThanAndEndTimeGreaterThanLastOneDay(functionId,
+                lastNotifiedAnomaly);
+        if (filteredAnomalies != null && filteredAnomalies.size() > 0) {
+          mergedAnomaliesAllResults.addAll(filteredAnomalies);
```
Comment (apucher) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/v2/AlertTaskRunnerV2.java]: addAll is idempotent for empty collections already

Comment (puneetjaiswal) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/v2/AlertTaskRunnerV2.java]: yeah, just a habit of doing both checks.

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/bao/MergedAnomalyResultManager.java (hunk 1 above)
Comment (puneetjaiswal) [inline thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/bao/MergedAnomalyResultManager.java]: find Not Notified anomalies ByFunctionId (given functionId) And anomaly Id Lesser Than (given anomaly id) And EndTimeGreaterThanLastOneDay

**Comment by apucher** [comment]
Also seems to fix #1140

**Pull request merged**